test/control_test.cpp

```diff
@@ -4,20 +4,6 @@
 
 using namespace devilution;
 
-TEST(Control, SetSpell)
-{
-	pnumlines = 1;
-	pSpell = SPL_FIREBOLT;
-	pSplType = RSPLTYPE_CHARGES;
-	auto &myPlayer = Players[MyPlayerId];
-	SetSpell();
-	EXPECT_EQ(spselflag, false);
-	EXPECT_EQ(myPlayer._pRSpell, SPL_FIREBOLT);
-	EXPECT_EQ(myPlayer._pRSplType, RSPLTYPE_CHARGES);
-	EXPECT_EQ(pnumlines, 0);
-	EXPECT_EQ(force_redraw, 255);
-}

```

Comment:
Can we replace this test with something equivalent instead of just removing it :thinking:

Reply:
I also thought about it.
We would need to set up the the resolution + player spell list + calculate the mouse position for this spell.

But then I saw what we (initially) test and I thought that's currently not worth it:

```
void SetSpell()
{
	spselflag = false;
	if (pSpell == SPL_INVALID) {
		return;
	}

	ClearPanel();

	auto &myPlayer = Players[MyPlayerId];
	myPlayer._pRSpell = pSpell;
	myPlayer._pRSplType = pSplType;

	force_redraw = 255;
}
```

One branch and three assignments.